ajax/widget/js/widget.js

```diff
@@ -0,0 +1,41 @@
+let employeeList = new XMLHttpRequest();
+employeeList.onreadystatechange = function () {
+    if (employeeList.readyState === 4) {
```

Comment:
I know you are following the courses but after you do the code try to review it for some obvious improvements. Looking at the code and getting a feel for how it flows will be invaluable over time.
So I would do a `if (employeeList.readyState !== 4) {return}`.
'Return first' reduces overall indentation which is always a bonus IF it does increase readability.

Reply:
Done. Will try to keep an eye out for something like this.